pkg/util/membercluster_client.go

```diff
@@ -132,37 +133,51 @@ func NewClusterClientSetForAgent(clusterName string, _ client.Client, clientOpti
 
 	if clusterConfig != nil {
 		if clientOption != nil {
-			clusterConfig.QPS = clientOption.QPS
-			clusterConfig.Burst = clientOption.Burst
+			if clientOption.RateLimiterGetter != nil {
+				clusterConfig.RateLimiter = clientOption.RateLimiterGetter(clusterName)
+			}
 		}
 		clusterClientSet.KubeClient = kubeclientset.NewForConfigOrDie(clusterConfig)
 	}
 	return &clusterClientSet, nil
 }
 
+// NewClusterDynamicClientSetFunc is a function that returns a dynamic client for the given member cluster.
+type NewClusterDynamicClientSetFunc = func(clusterName string, client client.Client, clientOption *ClientOption) (*DynamicClusterClient, error)
```

Comment:
What is the difference between removing the equal sign and defining the function variable directly and the current definition method?
`type NewClusterDynamicClientSetFunc func(clusterName string, client client.Client, clientOption *ClientOption) (*DynamicClusterClient, error)`